server/@types/dto/index.d.ts

```diff
@@ -208,4 +208,9 @@ declare module 'inductionDto' {
     workTypeOther?: string
     role?: string
   }
+
+  export interface EducationDto extends ReferencedAndAuditable {
```

Comment:
I'm in two minds about whether this should be a "DTO" or a "view model"
(in other places for similar things we've used the concept of a view model, but I guess you could argue a "view model" is a DTO, and we'll be wanting a DTO much like this when we get into the Create Education stories ... 🤔 )

Reply:
Let's leave it as a DTO for now .... though it might eat me alive and we might come back to it! 🤣